### PR TITLE
docs: add bi-weekly report (1120-1203)

### PR DESCRIPTION
**Pull request by saltism**
Fixes: #[Add issue number here]

Changes:

Screenshots of the change:

### CODE REVIEW

File: blog/zh/blog/2023/12/04/bi-weekly-report.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+## 近期亮点功能
+
+- [新增 `multi-auth` 插件](https://github.com/apache/apisix/pull/10482)（贡献者：[madhawa-gunasekara](https://github.com/madhawa-gunasekara))
+
+- [`openid-connect` 新增配置项 "required scopes"](https://github.com/apache/apisix/pull/10493)（贡献者：[csotiriou](https://github.com/csotiriou))
+
+- [增强 `body-transformer` 插件](https://github.com/apache/apisix/pull/10496)（贡献者：[yongxiaodong](https://github.com/yongxiaodong))
+
+## 最新博客速览
```
Comment (Yilialinn) [inline blog/zh/blog/2023/12/04/bi-weekly-report.md:41]: Replace the oldest one with https://apisix.apache.org/zh/blog/2023/11/30/migu-video-utilizes-apisix/

File: blog/en/blog/2023/12/04/bi-weekly-report.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+From 11.20 to 12.03, a total of 22 contributors made 45 commits to Apache APISIX. We sincerely appreciate your contributions to Apache APISIX.
+
+Our bi-weekly Apache APISIX community report is your window into the project's weekly developments. It is a tool to facilitate your seamless integration into the Apache APISIX community, ensuring that you stay well-informed and actively involved.
+
+## Contributor Statistics
+
+![Apache APISIX Contributors List](https://static.apiseven.com/uploads/2023/12/04/PnNWmVdX_1204-Con.png)
+
+![Apache APISIX New Contributors](https://static.apiseven.com/uploads/2023/12/04/vrpTk1y4_1204-New.png)
```
Comment (Yilialinn) [inline blog/en/blog/2023/12/04/bi-weekly-report.md:32]: This picture can't show properly. Can you see it? @JYan00

File: blog/zh/blog/2023/12/04/bi-weekly-report.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+从 2023.11.20 至 2023.12.03，有 22 名开发者提交了 45 个 commit，为 Apache APISIX 做出了重要贡献。感谢这些伙伴们对 Apache APISIX 的无私支持！正是因为你们的付出，才能让 Apache APISIX 项目不断改进、提升和壮大。
+
+Apache APISIX 社区的双周报旨在帮助社区成员更全面地了解社区的最新动态，方便大家参与到 Apache APISIX 社区中来。
+
+## 贡献者统计
+
+![贡献者名单](https://static.apiseven.com/uploads/2023/12/04/PnNWmVdX_1204-Con.png)
+
+![新晋贡献者](https://static.apiseven.com/uploads/2023/12/04/vrpTk1y4_1204-New.png)
```
Comment (Yilialinn) [inline blog/zh/blog/2023/12/04/bi-weekly-report.md:31]: Ditto

File: blog/zh/blog/2023/12/04/bi-weekly-report.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+title: "社区双周报 (11.20 - 12.03)"
+keywords: ["Apache APISIX", "API 网关", "社区周报", "贡献者"]
+description: Apache APISIX 社区的双周报旨在帮助社区成员更全面地了解社区的最新动态，方便大家参与到 Apache APISIX 社区中来。
+tags: [Community]
+image: https://static.apiseven.com/uploads/2023/12/04/ZgiF2B5y_1204-ZH.png
+---
+> 最近，我们修复并改进了 Apache APISIX 的部分功能，如新增 `multi-auth` 插件、`openid-connect` 新增配置项 "required scopes"、增强 `body-transformer` 插件。详情请阅读本期双周报。
+<!--truncate-->
+
+## 导语
```
Comment (Yilialinn) [inline blog/zh/blog/2023/12/04/bi-weekly-report.md:11]: Shall we add a conclusion of new features here like before? If there is only one new feature, then we can delete it.

File: blog/zh/blog/2023/12/04/bi-weekly-report.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+Apache APISIX 项目始终秉承着开源社区协作的精神，自问世起便崭露头角，如今已经成为全球最活跃的开源 API 网关项目之一。正如谚语所言，“众人拾柴火焰高”，这一辉煌成就，得益于整个社区伙伴的协同努力。
+
+从 2023.11.20 至 2023.12.03，有 22 名开发者提交了 45 个 commit，为 Apache APISIX 做出了重要贡献。感谢这些伙伴们对 Apache APISIX 的无私支持！正是因为你们的付出，才能让 Apache APISIX 项目不断改进、提升和壮大。
+
+最近，我们对一些功能进行了修复和改进，更新内容总结如下：
```
Comment (Yilialinn) [inline blog/zh/blog/2023/12/04/bi-weekly-report.md]:
```suggestion
最近，我们对一些功能进行了新增和改进，更新内容总结如下：
```

File: blog/en/blog/2023/12/04/bi-weekly-report.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
+---
+title: "Biweekly Report (November 20 - December 03)"
+keywords: ["Apache APISIX", "API Gateway", "Weekly Report", "Contributor"]
+description: Our bi-weekly Apache APISIX community report is your window into the project's weekly developments. It is a tool to facilitate your seamless integration into the Apache APISIX community, ensuring that you stay well-informed and actively involved.
+tags: [Community]
+image: https://static.apiseven.com/uploads/2023/12/04/ZD3OVqNc_1204-ENG.png
+---
+
+> We have recently made some fixes and improvements to specific features within Apache APISIX. The updates include adding a `multi-auth` plugin, adding the required scopes configuration property to `openid-connect` plugin, and enhancing the `body-transformer` plugin. For additional information, please consult the bi-weekly report.
```
Comment (Yilialinn) [inline blog/en/blog/2023/12/04/bi-weekly-report.md]:
```suggestion
> We have recently made some additions and improvements to specific features within Apache APISIX. The updates include adding a `multi-auth` plugin, adding the required scopes configuration property to `openid-connect` plugin, and enhancing the `body-transformer` plugin. For additional information, please consult the bi-weekly report.
```

File: blog/zh/blog/2023/12/04/bi-weekly-report.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
+---
+title: "社区双周报 (11.20 - 12.03)"
+keywords: ["Apache APISIX", "API 网关", "社区周报", "贡献者"]
+description: Apache APISIX 社区的双周报旨在帮助社区成员更全面地了解社区的最新动态，方便大家参与到 Apache APISIX 社区中来。
+tags: [Community]
+image: https://static.apiseven.com/uploads/2023/12/04/ZgiF2B5y_1204-ZH.png
+---
+> 最近，我们修复并改进了 Apache APISIX 的部分功能，如新增 `multi-auth` 插件、`openid-connect` 新增配置项 "required scopes"、增强 `body-transformer` 插件。详情请阅读本期双周报。
```
Comment (Yilialinn) [inline blog/zh/blog/2023/12/04/bi-weekly-report.md]:
```suggestion
> 最近，我们新增并改进了 Apache APISIX 的部分功能，如新增 `multi-auth` 插件、`openid-connect` 新增配置项 "required scopes"、增强 `body-transformer` 插件。详情请阅读本期双周报。
```

File: blog/en/blog/2023/12/04/bi-weekly-report.md [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+<!--truncate-->
+
+## Introduction
+
+From its inception, the Apache APISIX project has embraced the ethos of open-source community collaboration, propelling it into the ranks of the most active global open-source API gateway projects. The proverbial wisdom of 'Many hands make light work' rings true in our way, made possible by the collective dedication of our community.
+
+From 11.20 to 12.03, a total of 22 contributors made 45 commits to Apache APISIX. We sincerely appreciate your contributions to Apache APISIX.
+
+We have recently addressed and enhanced various features, and here is a summary of the updates:
```
Comment (Yilialinn) [inline blog/en/blog/2023/12/04/bi-weekly-report.md]:
```suggestion
We have recently added and enhanced several plugins, and here is a summary of the updates:
```

**Review by SkyeYoung — APPROVED** [review]

**Review by bzp2010 — APPROVED** [review]

**Review by guoqqqi — APPROVED** [review]

**Pull request merged**